### ISSUE TITLE
training the vocoder

### ISSUE THREAD

**Issue by juanalonso** [Kinyugo/msanii#8]
Hi! Once again, great work.

Can you share any recommendations for training the vocoder? 
Should the audio files be prepared in any way? (trimmed to 11 seconds, normalized...)
How many minutes / hours of audio do you recommend to trin the vocoder?

Thank you very much

**Comment by Kinyugo** [comment]
Hello. 

There are not limits to the sizes of the files, trimming and padding happens automatically to match the `num_frames` parameter. I trained was trained on roughly 908 files totalling to about 63hrs for 40k steps. Check the appendix section of the paper or the config file for more parameters.

However, there is a limitation with the dataloader as it excepts a folder with all the audio files at the top level. So sub-folders won't work. 

Feel free to reach out in-case of any other questions.
